src/host/node_connections.h

```diff
@@ -247,18 +247,22 @@ namespace asynchost
               serialized::overlay<consensus::AppendEntriesIndex>(p, psize);
             // Find the total frame size, and write it along with the header.
             auto count = ae.idx - ae.prev_idx;
-            uint32_t frame = (uint32_t)(
-              size_to_send +
-              ledger.framed_entries_size(ae.prev_idx + 1, ae.idx));
+
+            auto framed_entries =
```

Comment:
Removed the `framed_entries_size()` ledger API as the frame size can be deduced from the vector returned by `read_framed_entries()`.